tests/test_science_utils.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import json
+from pandas import read_csv
+from numpy.testing import assert_allclose
+from gnpy.core.info import create_input_spectral_information
+from gnpy.core.elements import RamanFiber
+from gnpy.core.network import load_sim_params
+
+#@pytest.mark.parametrize()
+def test_raman_fiber():
+    # spectral information generation
+    power = 1e-3
+    with open('tests/data/eqpt_config.json', 'r') as file:
```

Comment:
other tests appear to support relative paths better, i.e. like this:

```py
from pathlib import Path
TEST_DIR = Path(__file__).parent

with open(TEST_DIR / 'tests' / 'data' / 'eqpt_config.json'):
   ...
```

The biggest advantage is the `__file__` statement which make this work even when running from inside another directory.

Reply:
The commit 03168ba should fix it.